### ISSUE TITLE
Variable interpretation not working.

### ISSUE THREAD

**Issue by ErikEvenson** [hashicorp/nomad#792]
Nomad (v0.2.3) variable interpretation per the docs does not work:

			env {
				NOMAD_CLASS = "$nomad.class"
			}

gives

```
$ env
NODE_VERSION=5.5.0
HOSTNAME=9fe465e33db8
NOMAD_TASK_DIR=/local
NPM_CONFIG_LOGLEVEL=info
user=node
LS_COLORS=
NOMAD_CLASS=$nomad.class
NOMAD_IP=192.168.99.200
NOMAD_CPU_LIMIT=500
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
PWD=/home/node
NOMAD_ALLOC_DIR=/alloc
NOMAD_MEMORY_LIMIT=256
SHLVL=1
HOME=/home/node
NOMAD_PORT_web=8080
_=/usr/bin/env
```

**Comment by let4be** [comment]
@ErikEvenson this is going to be fixed in 0.3 :)
see https://github.com/hashicorp/nomad/issues/781

**Comment by ErikEvenson** [comment]
OK -- I thought it was in 0.2.3 since it was in the current docs.  I will close.  Cheers.
